docs/overrides/main.html

```diff
@@ -12,16 +12,20 @@
     }
   </style>
   <div class="announcement">
-
+  <!--
     <h2><b>THIS SITE IS UNDER DEVELOPMENT</b><br></h2>
 
     <h3>
       We encourage feedback via the giscus form on the bottom of the main page. <br>
     </h3>
-
-<!--
-    <h2>BIDS steering group election</h2>
-
+  -->
+    <h2><b>BIDS steering group election</b></h2>
+    <p>This year's election has started.     
+    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
+    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
+    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
+    </p>
```

Comment:
```suggestion
    <p>This year's election has started.     
    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
    </p>    
    <p>    
    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
    </p>
```

Reply:
to avoid very long line

![image](https://github.com/user-attachments/assets/b3a21903-71b4-4e70-803d-0cac4271b3cd)